org.envirocar.app/src/org/envirocar/app/views/carselection/CarSelectionListAdapter.java

```diff
@@ -150,37 +156,37 @@ public View getView(int position, View convertView, ViewGroup parent) {
         }
 
-        // Set the onClickListener for a single row.
-        convertView.setOnClickListener(v -> new MaterialDialog.Builder(mContext)
-                .items(state)
-                .itemsCallback((materialDialog, view, i, charSequence) -> {
-                    switch (i) {
-                        case 0:
-                            // Call the callback
-                            mCallback.onDeleteCar(car, mSelectedButton);
-                            break;
-                        case 1:
-                            if(car.equals(mSelectedCar))
-                                return;
-
-                            // Uncheck the currently checked car.
-                            if (mSelectedButton != null) {
-                                mSelectedButton.setChecked(false);
-                            }
-
-                            // Set the new car as selected car type.
-                            mSelectedCar = car;
-                            mSelectedButton = tmpHolder.mRadioButton;
-                            mSelectedButton.setChecked(true);
-
-                            // Call the callback in order to react accordingly.
-                            mCallback.onSelectCar(car);
-                            break;
-                        default:
-                            LOG.warn("No action selected!");
-                    }
-                })
-                .show());
+//        // Set the onClickListener for a single row.
+//        convertView.setOnClickListener(v -> new MaterialDialog.Builder(mContext)
+//                .items(state)
+//                .itemsCallback((materialDialog, view, i, charSequence) -> {
+//                    switch (i) {
+//                        case 0:
+//                            // Call the callback
+//                            mCallback.onDeleteCar(car, mSelectedButton);
+//                            break;
+//                        case 1:
+//                            if(car.equals(mSelectedCar))
+//                                return;
+//
+//                            // Uncheck the currently checked car.
+//                            if (mSelectedButton != null) {
+//                                mSelectedButton.setChecked(false);
+//                            }
+//
+//                            // Set the new car as selected car type.
+//                            mSelectedCar = car;
+//                            mSelectedButton = tmpHolder.mRadioButton;
+//                            mSelectedButton.setChecked(true);
+//
+//                            // Call the callback in order to react accordingly.
+//                            mCallback.onSelectCar(car);
+//                            break;
+//                        default:
+//                            LOG.warn("No action selected!");
+//                    }
+//                })
+//                .show());
```

Comment:
If you feel that this section is not necessary any more, just delete it and do not comment it out. However, I'd prefer to keep the context menu as an optional function for deleting or selecting a car beside the button based functions.